src/components/carousel/index.jsx

```diff
@@ -182,6 +217,28 @@ const Carousel = ({
 		<DefaultExitFullScreenButton id={id} onClick={toggleFullScreen} />
 	);
 
+	const resolvedStartAutoplayButton = startAutoplayButton ? (
+		resolvedButton(
+			startAutoplayButton,
+			id,
+			`${COMPONENT_CLASS_NAME}__button--start-auto-play`,
+			toggleAutoplay
+		)
+	) : (
+		<DefaultStartAutoplayButton id={id} onClick={toggleAutoplay} />
+	);
+
+	const resolvedStopAutoplayButton = stopAutoplayButton ? (
+		resolvedButton(
+			stopAutoplayButton,
+			id,
+			`${COMPONENT_CLASS_NAME}__button--stop-auto-play`,
+			toggleAutoplay
+		)
+	) : (
+		<DefaultStopAutoplayButton id={id} onClick={toggleAutoplay} />
+	);
+
 	/* istanbul ignore next  */
 	const fullScreenEnabledAllowed =
 		(document.fullscreenEnabled || document.webkitFullscreenEnabled) && enableFullScreen;
```

Comment:
`document` is not available on Fusion server side render and will throw an error as per the `window` reference above

Reply:
@matthewroach updated. thanks for the note